lib/spack/spack/object_file/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+"""
+This package includes parsing and modification utilities for object files.
+Currently only ELF files.
+"""
```

Comment:
(question) what other types of files would you anticipate parsing? Windows format I would assume. Something else too?

Reply:
There's also COFF, although probably not widely used. And macho for macos, although I noticed we have a vendored dep for that.